tox.ini

```diff
@@ -3,20 +3,17 @@ linting_folders=src/wagtailbakery/ tests/ examples/
 
 [tox]
 envlist=
-  py310-django{32}-wagtail215,                     # Wagtail 2.15 LTS - latest supported Python - Django LTS
-  py{38,39,310}-django41-wagtail40,                # Wagtail 4.0 - all Python - latest Django
-  py310-django{32,40,41}-wagtail41,                # Wagtail 4.1 LTS - latest Python - all Django
-  wagtailmain                                      # Wagtail main latest compatible version
+  py{38,39,310,311}-django{32,40,41}-wagtail{41,42} # Wagtail 4.1 LTS and 4.2 pre-release - all supported Python - all Django
+  wagtailmain                                       # Wagtail main latest compatible version
 
 [testenv]
 commands=py.test --cov=wagtailbakery --cov-report=xml {posargs}
 deps=
-  django32: django>=3.2,<3.3       # WT 2.15 LTS
-  django40: django>=4.0,<4.1       # WT 4.0, WT 4.1 LTS
-  django41: django>=4.1,<4.2       # WT 4.0, WT 4.1 LTS
-  wagtail215: wagtail>=2.15,<2.16  # Old LTS
-  wagtail40: wagtail>=4.0,<4.1  # Current latest
+  django32: django>=3.2,<3.3
+  django40: django>=4.0,<4.1
+  django41: django>=4.1,<4.2
   wagtail41: wagtail>=4.1rc1,<4.2  # Current LTS
```

Comment:
```suggestion
  wagtail41: wagtail>=4.1,<4.2  # Current LTS
```

Reply:
Thanks, this is now fixed in the latest commit https://github.com/wagtail/wagtail-bakery/pull/79/commits/6ce7b17599914c80a919729c3c10a738593b7b4a
as well as the one below.